RPI/RpiDemo/RpiDemoComponentImpl.cpp

```diff
@@ -129,7 +129,7 @@ namespace Rpi {
   {
       // convert incoming data to string. If it is not printable, set character to '*'
       char uMsg[serBuffer.getSize()+1];
-      char* bPtr = (char*)serBuffer.getData();
+      char* bPtr = reinterpret_cast<char*>(serBuffer.getData());
```

Comment:
`CHAR*`

Reply:
?

Reply:
I think the conversion of `char` to `CHAR` should be done in another PR at some point. If I used CHAR here I'd want to update the entire file to use it.

Reply:
Agreed.  I had erroneously thought this change went through already.